manifests/09-prometheus-rules.yaml

```
@@ -17,3 +17,7 @@ spec:
         message: |
           Image Registry Storage configuration has changed in the last 30
           minutes. This change may have caused data loss.
+  - name: image-registry.rules
+    rules:
+    - expr: sum(apiserver_v1_image_imports_total)
+      record: apiserver_v1_image_imports_total:sum
```

To comply with [best practices](https://prometheus.io/docs/practices/rules/):

```suggestion
      record: :apiserver_v1_image_imports:sum
```